rapids-cmake/test/generate_resource_spec.cmake

```diff
@@ -73,7 +73,7 @@ function(rapids_test_generate_resource_spec DESTINATION filepath)
     if(CUDAToolkit_FOUND)
       set(compile_options "-I${CUDAToolkit_INCLUDE_DIRS}" "-DHAVE_CUDA")
     endif()
-    set(link_options ${CUDA_cudart_LIBRARY})
+    set(link_options ${CUDA_cudart_LIBRARY} "-lpthread -lrt -ldl")
```

Comment:
This shouldn't be quoted - `execute_process()` will treat `"-lpthread -lrt -ldl"` as a single argument.

Reply:
So should it be this?
```
set(link_options ${CUDA_cudart_LIBRARY} -lpthread -lrt -ldl)
```

Reply:
Yes, exactly.

Reply:
I was following the pattern from https://github.com/rapidsai/rapids-cmake/pull/516/files#diff-92f51ddffa5e81afb760f384afc6f2249d2b75ed468d9f4fbaa77bc8560a8ed1R74
```
set(compile_options "-I${CUDAToolkit_INCLUDE_DIRS}" "-DHAVE_CUDA")
```

Reply:
Note that in that example there are quotes surrounding each argument, not the whole argument list. The equivalent here would be:

```cmake
set(link_options ${CUDA_cudart_LIBRARY} "-lpthread" "-lrt" "-ldl")
```